roles/metrics_server/templates/metrics-server-viewing.j2

```diff
@@ -0,0 +1,41 @@
+# permissions for viewing metrics -- this isn't necessarily tied to metrics-server directly,
+# but we don't really have a better place for it right now
+
+kind: ClusterRole
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+  name: system:openshift:aggregate-to-view:resource-metrics
+rules:
+- apiGroups:
+  - metrics.k8s.io
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+- apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
This doesn't look like it will work, also this doesn't need to be a template as there is no templating.

Reply:
The definitions were taken directly from the output of the previous PR, and the role logic should glob all files starting with `metrics-server-*.js`, IIRC